lecli/log_api.py

```diff
@@ -0,0 +1,174 @@
+"""
+Log API module.
+"""
+import json
+import requests
+
+from lecli import api_utils
+from lecli import response_utils
+
+def _url():
+    """
+    Get rest query url of log resource id.
+    """
+    return 'https://rest.logentries.com/management/logs'
```

Comment:
Should we do this for all of the endpoints?

At the moment, they're all hard-coded.